server/Assignments_filler/‏‏DB_call.py

```diff
@@ -0,0 +1,50 @@
+import logging
+import os
+
+import mysql.connector
+from mysql.connector import Error
+from mysql.connector import errorcode
+
+# db_host = os.environ['DIGITAL_OCEAN_HOST']
+# db_name = os.environ['DATABASE_NAME']
+# db_user = os.environ['DO_DB_USER']
+# db_pass = os.environ['DO_DB_PASSWORD']
+# db_port = os.environ['DO_DB_PORT']
+
+db_host = "host"
+db_name = "name"
+db_user = "user"
+db_pass = "password"
+db_port = "port"
+
+logging.basicConfig(level=logging.INFO, format="%(asctime)s [%(levelname)s]: %(message)s",
+                    handlers=[logging.FileHandler("filler.log"), logging.StreamHandler()])
+
+def connect_db():
+    return mysql.connector.connect(host=db_host, database=db_name, port=db_port, user=db_user, password=db_pass)
```

Comment:
This should resolve the cache issue. In MySQL if you don't commit, next time you run the same query, it will give you the same result.

Doesn't seem to fix the issue at hand though, I think there is something with comparing if there is a new comment broken as well.
```suggestion
    connection = mysql.connector.connect(host=db_host, database=db_name, port=db_port, user=db_user, password=db_pass)
    connection.autocommit = True
    return connection
```